src/pages/index.js

```diff
@@ -28,44 +30,146 @@ const enableValidation = (config) => {
 
 enableValidation(config);
 
-//Create Card Section
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createNewCard(cardData, "#card-template");
-      cardSection.addItem(cardElement);
-    },
+//Create API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "0e6b8e99-3fb6-490c-b2a5-5d193b29415c",
+    "Content-Type": "application/json",
   },
-  "elements__list"
-);
+});
 
-cardSection.renderItems();
+//Card Actions
+let cardSection;
+
+api
+  .getInitialCards()
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all